### PR TITLE
lib.sh: install_yq fails on ppc64le

### PR DESCRIPTION
**Pull request by nitkon**
Fixes: #124

Signed-off-by: Nitesh Konkar niteshkonkar@in.ibm.com

### CODE REVIEW

**Comment by jodh-intel** [comment]
I agree with @grahamwhaley. Please could you update and re-push @nitkon?

**Comment by nitkon** [comment]
@grahamwhaley  @jodh-intel : Thanks for the inputs. Updated and re-pushed.

File: .ci/lib.sh [hunk 1]
```diff
@@ -40,6 +40,10 @@ install_yq() {
 	"x86_64")
 		goarch=amd64
 		;;
+
+	"ppc64le")
+		goarch=ppc64le
+                ;;
```
Comment (grahamwhaley) [inline .ci/lib.sh]: you're still missing a blank line below this one to match the other case statements ;-)

Comment (nitkon) [reply .ci/lib.sh]: Okay, I realised that before sending the original patch, but for some reason, I was just maintaining what was already there. 
https://github.com/kata-containers/packaging/blob/7705d68f7dadc189c3b836ad5972056ab8e881fb/.ci/lib.sh#L43

Comment (nitkon) [reply .ci/lib.sh]: I shall resend the patch.

**Comment by grahamwhaley** [comment]
thx @nitkon - yeah, I know it was not all your whitespace stuff - but, whilst you are there ;-) thx!

**Comment by nitkon** [comment]
@grahamwhaley : No problem at all! :-) Re-pushed!

File: .ci/lib.sh (hunk 1 above)
Comment (jodh-intel) [reply .ci/lib.sh]: I hate to say this, but this is *still* inconsistent with what is already there. The two semi-colons aren't aligned like they are in the other two cases.

Comment (nitkon) [reply .ci/lib.sh]: @jodh-intel : Now? :-(

**Comment by jodh-intel** [comment]
Thanks @nitkon.

lgtm

**Comment by nitkon** [comment]
All green. Good to merge.

**Pull request merged**